tests/unit/local/docker/test_container.py

```diff
@@ -507,102 +421,6 @@ def test_must_not_support_input_data(self):
             self.container.start(input_data="some input data")
 
 
-class TestContainer_wait_for_result(TestCase):
```

Comment:
Why are all these tests removed?

Reply:
I refactored the Container && LambdaContainer classes, so I moved some test cases based on this refactoring